boreas/ping.c

```diff
@@ -144,8 +144,8 @@ send_icmp_v6 (int soc, struct in6_addr *dst, int type)
   icmp6 = (struct icmp6_hdr *) sendbuf;
   icmp6->icmp6_type = type; /* ND_NEIGHBOR_SOLICIT or ICMP6_ECHO_REQUEST */
   icmp6->icmp6_code = 0;
-  icmp6->icmp6_id = 234;
-  icmp6->icmp6_seq = 0;
+  icmp6->icmp6_id = rand ();
```

Comment:
`max(rand (), 1)` ?

That way we can't accidentally get 0 if rand () returns 0? (I'm assuming that rand here can return 0, but again I'm not much of a C developer :P

Reply:
Thanks for the review!